config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbinstances.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -211,21 +215,26 @@ spec:
                   nodeCount:
                     description: Read capacity, i.e. number of nodes in a read pool
                       instance.
+                    format: int32
                     type: integer
                 type: object
               resourceID:
                 description: Immutable. Optional. The instanceId of the resource.
-                  Used for creation and acquisition. When unset, the value of `metadata.name`
-                  is used as the default.
+                  If not given, the metadata.name will be used.
                 type: string
+                x-kubernetes-validations:
+                - message: ResourceID field is immutable
+                  rule: self == oldSelf
```

Comment:
let's remove this CEL validation check, it has been guardrailed by the NewAlloyDBIdentity fn.

Reply:
I'm a bit confused about whether we should add this validation or not in general. It looks like this validation still exists in the template: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/dev/tools/controllerbuilder/template/apis/types.go#L40. Should this line be removed from the template?

Reply:
Yes, I think so. Please go ahead and move it from the template

Reply:
Removed it from AlloyDBInstance.